packages/big-design/src/components/TableNext/DataCell/styled.tsx

```diff
@@ -43,6 +43,17 @@ export const StyledTableDataCell = styled.td<DataCellProps>`
       vertical-align: ${verticalAlign};
     `};
 
+  ${({ checkEmptyCell }) => {
+    return (
+      checkEmptyCell &&
+      css`
+        &:empty {
+          display: none;
+        }
+      `
+    );
+  }};
```

Comment:
🍹 I'm not very familiar with this. Could you explain the need for this?

Reply:
Sure, so if the HelperRowComponent returns null, then the cell is gonna be empty. So in order to disappear the cell, I had to set this css logic. 

```tsx
 const HelperRow = ({ parentRowIndex }) => {
  
    if (state[parentIndex].isLoading) {
       return <LoadingIcon/>
    }

   if (state[parentIndex] !== undefined && state[parentIndex].remaining) {

     const itemsToView = state[parentIndex].remaining;

     return <Button onClick={() => viewMore(parentRowIndex)} />
   }

   return null
  };
```